sass/directives/_gradients.scss

```diff
@@ -106,6 +106,14 @@ $gradient_var_list: (
   background-attachment: fixed;
 }
 
+@mixin imageGradient($img-url, $dark, $medium, $light) {
```

Comment:
In the future I'd want to refactor this so that it's more like the mixin on line 92 + image, and the 3 different transparencies are all arguments

On line 92 and the other mixins, we can probably [get rid of the fallbacks](http://caniuse.com/#search=gradient) and also refactor so that 3 transparencies can be passed in

but this is good for now!